Backpack/src/main/java/net/skyscanner/backpack/util/BpkViewPumpContextWrapper.kt

```diff
@@ -0,0 +1,19 @@
+@file:JvmName("BpkViewPumpContextWrapper")
+
+package net.skyscanner.backpack.util
+
+import android.content.Context
+import android.content.ContextWrapper
+import io.github.inflationx.viewpump.ViewPumpContextWrapper
+
+class BpkViewPumpContextWrapper {
+  companion object {
```

Comment:
Sound like you don't need any state in this class stored but only using it to call a static function -- why not use an `object` instead?